### PR TITLE
Fixes #235 changed code of conduct text and link to participation guildlines

### LINKED ISSUE #235: Add link to Participation Guidelines in footer
- [x] We'll need an icon 
 - [ ] Link should read "Participation Guidelines," should go between the Facebook and the License items, and the target should be https://www.mozilla.org/en-US/about/governance/policies/participation/

### PR DESCRIPTION
**Pull request by jayyyin**
do note I have changed the Class name here so this issue https://github.com/mozilla/mofo-bootstrap/issues/122 has to be fixed first before we pull this

### CODE REVIEW

**Comment by jayyyin** [comment]
The issue this fixes is here https://github.com/mozilla/foundation.mozilla.org/issues/235

File: source/pug/templates/master.pug [hunk 1]
```diff
@@ -78,8 +78,8 @@ html
               a(href="https://www.facebook.com/mozilla" class="footer-link-facebook") Facebook
             li.col-auto
               a(href="https://creativecommons.org/licenses/by/4.0" class="footer-link-cc-license") License
-            //- li.col-auto
-            //-   a(href="#" class="footer-link-code-of-conduct") Code of Conduct
+            - li.col-auto
```
Comment (gideonthomas) [inline source/pug/templates/master.pug]: remove the `-` on both lines

File: source/pug/templates/master.pug [hunk 2]
```diff
@@ -78,8 +78,8 @@ html
...
+            -   a(href="https://www.mozilla.org/en-US/about/governance/policies/participation/" class="footer-link-participation-guidelines") Participation Guidelines
```
Comment (gideonthomas) [inline source/pug/templates/master.pug]: remove the `/en-US` part of the URL.

Comment (gideonthomas) [reply source/pug/templates/master.pug]: also, let's leave the class name as what it was before.

When we land your other PR, we will need to update the version of mofo-bootstrap here so we can change the class name here in that patch.

Comment (jayyyin) [reply source/pug/templates/master.pug]: ok

**Comment by jayyyin** [comment]
@gideonthomas alright so I've removed the en-gb part of the url and reverted the class name for now, when we get around to updating the required package I'll make another issue/PR

**Comment by gideonthomas** [comment]
Thanks @jayyyin, this looks right to me. I'll let @gvn or @mmmavis merge it in if they're ok with it as well.

**Comment by gvn** [comment]
LGTM. Thanks!

**Comment by jayyyin** [comment]
@gvn what's LGTM mean?
@gideonthomas thanks for the answers for getting started on these issues.

**Comment by gvn** [comment]
"Looks good to me". People often write that when they approve code for merging, similar to "R+".

**Comment by jayyyin** [comment]
@gvn I see, thanks man

**Pull request merged**